src/iregnet_fit.cpp

```diff
@@ -184,31 +207,103 @@ fit_cpp(Rcpp::NumericMatrix X, Rcpp::NumericMatrix y,
   }
   scale = scale_init; log_scale = log(scale);
 
+  /* Sort the whole matrix by censoring type, get the sorted X and y
+   * & Switch the right method for target_compute_grad_response
+   */
+  if(transformed_dist != IREG_DIST_EXTREME_VALUE){
```

Comment:
Does this mean that we are using the old code for the extreme value distribution?